### PR TITLE
RN_updates_2-40.1

### PR DESCRIPTION
**Pull request by rasika-chivate**
update the relese notes for PMM 2.40.1

### CODE REVIEW

File: docs/release-notes/2.40.1.md [hunk 1]
```diff
@@ -11,18 +11,25 @@ Percona Monitoring and Management (PMM) is an open-source database monitoring, m
...
-For instructions on upgrading to PMM 2.40.1 and mitigating issues from PMM 2.40.0, follow the steps below:
+!!! caution alert alert-warning "Important"
+    The steps below are only applicable to users who have upgraded to 2.40.0 and have been experiencing this problem. 
+
+To upgrade to PMM 2.40.1:
 
 1. Copy the file from `/srv/backup/grafana/grafana.db` to `/srv/grafana/grafana.db`
 
+    !!! caution alert alert-warning "Caution"
+        This step will cause any changes made to Grafana after upgrading to version 2.40.0 to be lost.
```
Comment (browseman) [inline docs/release-notes/2.40.1.md:27]: `This step will cause a loss of all changes made to Grafana after upgrading to version 2.40.0.`
Please consider this one, at least for me it "sound" better.

File: docs/release-notes/2.40.1.md [hunk 2]
```diff
@@ -11,18 +11,25 @@ Percona Monitoring and Management (PMM) is an open-source database monitoring, m
...
 - [PMM-12587](https://jira.percona.com/browse/PMM-12587) - After upgrading to PMM 2.40.0, some users may experience incorrect mappings between dashboards, folders, users, and groups. This can result in either a successful upgrade or a `500 internal server error`. The issue has now been resolved.
-- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the base OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
+- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
 
 
 ## How to upgrade to PMM 2.40.1
 
-For instructions on upgrading to PMM 2.40.1 and mitigating issues from PMM 2.40.0, follow the steps below:
+!!! caution alert alert-warning "Important"
+    The steps below are only applicable to users who have upgraded to 2.40.0 and have been experiencing this problem.
```
Comment (browseman) [inline docs/release-notes/2.40.1.md:20]: `experiencing this problem. `
Could you please specify explicitly for which problems this fix is needed: PMM-12592 + PMM-12576 + PMM-12587

**Review by browseman — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.40.1.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/release-notes/2.40.1.md]: Fixed.

File: docs/release-notes/2.40.1.md [hunk 3]
```diff
@@ -9,20 +9,31 @@ Percona Monitoring and Management (PMM) is an open-source database monitoring, m
 ## Fixed issues
 
 - [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working after upgrade to PMM 2.40.0.
-- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now.
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - After upgrading to PMM 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now.
```
Comment (rnovikovP) [inline docs/release-notes/2.40.1.md:12]: not sure this is needed. looks like typo

**Review by Nailya — APPROVED** [review]

File: docs/release-notes/2.40.1.md [hunk 4]
```diff
@@ -11,18 +11,28 @@ Percona Monitoring and Management (PMM) is an open-source database monitoring, m
 - [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working after upgrade to PMM 2.40.0.
 - [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now.
 - [PMM-12587](https://jira.percona.com/browse/PMM-12587) - After upgrading to PMM 2.40.0, some users may experience incorrect mappings between dashboards, folders, users, and groups. This can result in either a successful upgrade or a `500 internal server error`. The issue has now been resolved.
-- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the base OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
+- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
 
 
 ## How to upgrade to PMM 2.40.1
```
Comment (BupycHuk) [inline docs/release-notes/2.40.1.md:17]: I think it's better to rename this title as
```suggestion
## How to upgrade from 2.40.0 to PMM 2.40.1
```
to make it clear. Users might skip cautions below.

**Review by BupycHuk — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**